test/regressions/tests/TextField/FilledMultilineHiddenLabelTextField.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+
+export default function FilledMultilineHiddenLabelTextField() {
+  return <TextField variant="filled" multiline rows={3} hiddenLabel />;
```

Comment:
Should we:
- Add text inside the textbox to better visualize the spacing
- Reduce the numbers for rows from 3 to 1 to better visualize the spacing (remove one source of variance)
- Display a non multine hiddenLabel textbox side by side to make sure we have the right dimensions

?